src/coreclr/src/jit/lower.cpp

```diff
@@ -5910,6 +5957,73 @@ bool Lowering::NodesAreEquivalentLeaves(GenTree* tree1, GenTree* tree2)
     }
 }
 
+//------------------------------------------------------------------------
+// Lowering::CheckMultiRegLclVar: Check whether a MultiReg GT_LCL_VAR node can
+//                                remain a multi-reg.
+//
+// Arguments:
+//   lclNode     - the GT_LCL_VAR node
+//   retTypeDesc - a return type descriptor for the consuming node
```

Comment:
`retTypeDesc` can be both a user of this `LCL_VAR`: `(RET(LCL_VAR)` and the source `STORE_LCL_VAR(call)`, is it correct?

Reply:
Yes, it can come from a `GT_CALL` source or a `GT_RETURN` user.

Reply:
I've updated the comments.